iset.mm

```diff
@@ -150033,6 +150178,248 @@ the additive structure must be abelian (see ~ ringcom ), care must be
+      basfn sylancr unitssd ssexd mgpex ressplusgd unitmulcl csrg adantr simpr1
+      unitcld simpr2 simpr3 3jca ringass syldan 1unit simpr ringlidm cdsr coppr
+      w3a wbr isunitd mpbid dvdsr2d opprbasg opprring syl anbi12d reeanv simprl
+      ad2antrr dvdsrmuld simplll simplr syl13anc simprrl oveq1d syl3anc simprrr
+      opprmulg eqtr3d oveq2d 3eqtr3d syl2anc ringridm 3brtr3d breqtrd mpbir2and
+      eqtrd jca rexlimdvaa expimpd reximdv2 biimtrrid sylbid mpd isgrpde ) AHIZ
+      UAFUDBAUEJZCAUBJZXNABCBAUCJKZXNDLZCAUFJZBUGMKXNELZAUHZUIXNBXOXSCNNXTAXOXS
+      HXSOZXOOZUJXNBAPJZNXNPNUKANIYDNIZUOAHULYENAPAPUMUNUPXNYDABXNYDQXRYAUQURAX
+      SHYBUSUTAXOBUARZFRZDYCVAXNYFBIZYGBIZUDRZBIZVPZYFYDIZYGYDIZYJYDIZVPYFYGXOM
+      YJXOMYFYGYJXOMXOMKXNYLSZYMYNYOYPYDABYFYPYDQZXQYPDLZXNAVBIZYLYAVCZXNYHYIYK
+      VDVEYPYDABYGYQYRYTXNYHYIYKVFVEYPYDABYJYQYRYTXNYHYIYKVGVEVHYDAXOYFYGYJYDOZ
+      YCVIVJABXPDXPOZVKXNYHYMXPYFXOMYFKXNYHSZYDABYFUUCYDQZXQUUCDLZXNYSYHYAVCZXN
+      YHVLZVEZYDAXOXPYFUUAYCUUBVMVJUUCYFXPAVNJZVQZYFXPAVOJZVNJZVQZSZYGYFXOMZXPK
+      ZFBTZUUCYHUUNUUGUUCUUIAUUKBXPUULYFUUEUUCXPQUUCUUIQZUUCUUKQUUCUULQZUUFVRVS
+      UUCUUNUUPFYDTZGRZYFUUKUEJZMZXPKZGYDTZSZUUQUUCUUJUUTUUMUVEUUCFYDUUIAXOYFXP
+      UUDUURUUFUUCXOQUUHVTUUCGYDUULUUKUVBYFXPXNYDUUKPJKZYHYDAUUKHUUKOZUUAWAZVCU
+      USXNUUKVBIZYHXNUUKHIUVJAUUKUVHWBUUKUHWCZVCUUCUVBQUUHVTWDUVFUUPUVDSZGYDTZF
+      YDTUUCUUQUUPUVDFGYDYDWEUUCUVMUUPFYDBUUCYNUVMYIUUPSZUUCYNSZUVLUVNGYDUVOUVA
+      YDIZUVLSZSZYIUUPUVRYIYGXPUUIVQYGXPUULVQUVRUVAYFUVAXOMZYGXPUUIUVRYDUUIAXOU
+      VAYFUVRYDQUVRUUIQZUUCYSYNUVQUUFWGZUVRXOQUVOUVPUVLWFZUUCYMYNUVQUUHWGZWHUVR
+      XPUVAXOMZYGXPXOMZUVAYGUVRUUOUVAXOMZYGUVSXOMZUWDUWEUVRXNYNYMUVPUWFUWGKXNYH
+      YNUVQWIZUUCYNUVQWJZUWCUWBYDAXOYGYFUVAUUAYCVIWKUVRUUOXPUVAXOUVOUVPUUPUVDWL
+      ZWMUVRUVSXPYGXOUVRUVCUVSXPUVRXNUVPYHUVCUVSKUWHUWBUUCYHYNUVQUUGWGZYDAUVBXO
+      BUUKHYDUVAYFUUAYCUVHUVBOZWPWNUVOUVPUUPUVDWOWQZWRWSUVRXNUVPUWDUVAKUWHUWBYD
+      AXOXPUVAUUAYCUUBVMWTUVRXNYNUWEYGKUWHUWIYDAXOXPYGUUAYCUUBXAWTWSUWMXBUVRYGY
+      FYGUVBMZXPUULUVRYDUULUUKUVBYGYFUVRXNUVGUWHUVIWCUVRUULQZUVRXNUVJUWHUVKWCUV
+      RUVBQUWIUWCWHUVRUWNUUOXPUVRXNYHYNUWNUUOKUWHUWKUWIYDAUVBXOYDUUKHBYFYGUUAYC
+      UVHUWLWPWNUWJXEXCUVRUUIAUUKBXPUULYGXQUVRDLUVRXPQUVTUVRUUKQUWOUWAVRXDUWJXF
+      XGXHXIXJXKXLXM $.
+
+    $( The group of units of a commutative ring is abelian.  (Contributed by
+       Mario Carneiro, 19-Apr-2016.) $)
+    unitabl $p |- ( R e. CRing -> G e. Abel ) $=
+      ( ccrg wcel cgrp ccmn cabl crg crngring unitgrp syl cmgp cfv cvv wceq a1i
+      cbs cress co eqid crngmgp grpmndd basfn elex funfvex funfni sylancr eqidd
+      wfn cui csrg ringsrg unitssd ssexd subcmnd isabl sylanbrc ) AFGZCHGZCIGCJ
+      GVAAKGZVBALZABCDEMNZVABAOPZCQCVFBUAUBRVAESAVFVFUCUDVACVEUEVABATPZQVATQULA
+      QGVGQGZUFAFUGVHQATATUHUIUJVAVGABVAVGUKBAUMPRVADSVAVCAUNGVDAUONUPUQURCUSUT
+      $.
+
+    unitgrp.3 $e |- .1. = ( 1r ` R ) $.
+    $( The identity of the multiplicative group is ` 1r ` .  (Contributed by
```

Comment:
"The identity of the group of units of a ring is the ring unity."

Reply:
Changed in set.mm and iset.mm.

Reply:
I cannot see the changes...

Reply:
Ah, I see what I did wrong with git. Should be fixed now.

Thanks for checking.